easysid.cpp

```diff
@@ -65,10 +65,10 @@ EXPORT void set_pw(uint8_t voice, uint16_t pw)
 
 EXPORT void set_wave(uint8_t voice, uint8_t wave)
 {
-    if (voice < 1 || voice > 3) {
+    if (voice < 0 || voice > 2) {
         return;
     }
-    if (wave >= 4) {
+    if (wave > 3) {
```

Comment:
What's the purpose of this change? Since wave is an integer, these statements are equivalent, aren't they?